lib/Github/Api/Apps.php

```diff
@@ -137,16 +137,16 @@ public function removeInstallation($installationId)
      *
      * @link https://developer.github.com/v3/apps/installations/#list-repositories
      *
-     * @param int $userId
+     * @param array $params list of extra parameters.
      *
      * @return array
      */
-    public function listRepositories($userId = null)
+    public function listRepositories(array $params = [])
```

Comment:
We can't remove/replace parameters as this would be a BC break. If the user parameter is not required anymore this should be deprecated so that we can remove it in the next major version